src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private Random random = new Random();
+
+    public Color getRandomColor() {
+        Color[] color = Color.values();
+        int randomIndex = random.nextInt(color.length);
+        return color[randomIndex];
```

Comment:
randomIndex variable is reduntant.

```suggestion
        return color[random.nextInt(color.length)];
```